README.md

```diff
@@ -44,7 +43,6 @@ The `xcaddy` command will use the latest version of Caddy by default. You can cu
 
 As usual with `go` command, the `xcaddy` command will pass the `GOOS`, `GOARCH`, and `GOARM` environment variables through for cross-compilation.
 
-
```

Comment:
Also seems unnecessary (for example, I prefer the extra space between headers for navigability)